Dockerfile.livecd-iso-image

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN mkdir /root/image
+# You can get RHCOS releases from here https://releases-rhcos.cloud.privileged.psi.redhat.com/
+RUN curl -L -k https://releases-rhcos.cloud.privileged.psi.redhat.com/storage/releases/4.6-devel/46.82.202008181531-0/x86_64/rhcos-46.82.202008181531-0-live.x86_64.iso -o /root/image/livecd.iso
```

Comment:
should we hardcore the image version? also the URL seems to be internal?

Reply:
Yes, you can pxeboot the image. 

I don't know of any other public mirror for RHCOS images, if you have one we can use it. And wrt to hardcode the image, I'd say that this is a good starting point, then we should create some automation around it (a Makefile target maybe?) so we can set the image dynamically, but I'm open to suggestions here.

Reply:
than should we use the upstream here ? https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable

Reply:
or if you still want downstream releases, you can use https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/

Reply:
is there any PXE tests we can add (in this or follow up PR's)?

Reply:
Hey @ohadlevy I just pushed new changes, now the Dockerfile is dynamic and the build process has been included in the Makefile. The Readme has been updated as well.

I'm not sure how those PXE tests could be implemented, I can try if you give me some pointers.

Reply:
@mvazquezc thanks! in order to test it with PXE I would assume that we need some test-infra support:
1. build a cluster, generate an iso with injected ignition
2. extract iso artifices (kernel, initrd, live root)
now I could think of 3 different flows to test:
3a.  boot a vm directly (direct kernel/initrd boot) using output of 2
3b. setup a network, and run a dhcp/pxe container to serve output of 2 (I remember CNV had a easy container to test pxe).
3c. enhance the kexec script to use the output of 2.

HTH